### PR TITLE
Re org sig arch sub projects

### PR DESCRIPTION
**Pull request by dims**

### CODE REVIEW

File: sig-architecture/README.md [hunk 1]
```diff
@@ -33,28 +33,31 @@ The Chairs of the SIG run operations and processes governing the SIG.
...
-- **spartakus**
-  - Owners:
-    - https://raw.githubusercontent.com/kubernetes-incubator/spartakus/master/OWNERS
-- **steering**
+- **conformance-definition**
   - Owners:
-    - https://raw.githubusercontent.com/kubernetes/steering/master/OWNERS
-- **architecture-tracking**
+    - https://raw.githubusercontent.com/kubernetes/kubernetes/master/test/conformance/testdata/OWNERS
+    - https://github.com/orgs/kubernetes/teams/cncf-conformance-wg
```
Comment (bgrant0607) [inline sig-architecture/README.md]: Only OWNERS files should be in this list. That team can be in the README

File: sigs.yaml [hunk 2]
```diff
@@ -233,28 +233,32 @@ sigs:
...
+      - https://raw.githubusercontent.com/kubernetes-sigs/architecture-tracking/master/OWNERS
       - https://raw.githubusercontent.com/kubernetes/api/master/OWNERS
       - https://raw.githubusercontent.com/kubernetes/kubernetes/master/staging/src/k8s.io/api/OWNERS
-    - name: kubernetes-template-project
+    - name: conformance-definition
       owners:
-      - https://raw.githubusercontent.com/kubernetes/kubernetes-template-project/master/OWNERS
-    - name: spartakus
+      - https://raw.githubusercontent.com/kubernetes/kubernetes/master/test/conformance/testdata/OWNERS
+      - https://github.com/orgs/kubernetes/teams/cncf-conformance-wg
```
Comment (bgrant0607) [inline sigs.yaml]: Please remove this teams line

File: sig-architecture/README.md [hunk 3]
```diff
@@ -33,28 +33,31 @@ The Chairs of the SIG run operations and processes governing the SIG.
...
+    - https://raw.githubusercontent.com/kubernetes/contrib/master/OWNERS
     - https://raw.githubusercontent.com/kubernetes/utils/master/OWNERS
-- **contrib**
+    - https://raw.githubusercontent.com/kubernetes/kubernetes/master/vendor/OWNERS
+    - https://raw.githubusercontent.com/kubernetes/kubernetes/master/third_party/OWNERS
+    - https://raw.githubusercontent.com/kubernetes/kubernetes/master/staging/OWNERS
+- **steering**
   - Owners:
-    - https://raw.githubusercontent.com/kubernetes/contrib/master/OWNERS
+    - https://raw.githubusercontent.com/kubernetes/steering/master/OWNERS
```
Comment (jdumars) [inline sig-architecture/README.md]: Do we really own the steering committee?

**Comment by jdumars** [comment]
Thank you SO much for this Dims!

File: sig-architecture/README.md (hunk 3 above)
Comment (bgrant0607) [reply sig-architecture/README.md]: No. It's a workaround until Steering can have its own entry

**Pull request merged**